lib/helpers.js

```diff
@@ -224,6 +225,21 @@ exports.verifyDelegation = async ({keystoreId, host, capability}) => {
   return results;
 };
 
+exports.verifyHeaderValue = async (req, res, next) => {
```

Comment:
Let's be more explicit about which header value since this is no longer namespaced by the digest module but by only "helpers".

```suggestion
exports.verifyDigestHeaderValue = async (req, res, next) => {
```